spec/game/game_runner_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
-require 'spec_helper'
+
```

Comment:
I think you have a trailing newline here. It should have just been s red delete with nothing added